src/dom/document.ts

```diff
@@ -152,6 +152,16 @@ export class Document extends Node {
   get title(): string {
     return this.querySelector("title")?.textContent || "";
   }
+  set title(value: string) {
+    const head = this.head;
+    if (!head) return;
```

Comment:
So the thing is, you can update a title element if it exists even without a head element. It's just if both don't exist then you just ignore it.

Something like this:
```typescript
set title(value: string) {
  let titleElement = this.querySelector("title");
  if (!titleElement) {
    const { head } = this;
    if (!head) return;

    titleElement = this.createElement("title");
    head.appendChild(titleElement);
  }

  titleElement.textContent = value;
}
```